cduyhuynh/avamia
# Feat: call news api realtime

- Apply redis to store latest fetch time and news items
- Use redis to check if should fetch data or not to maintain quota limit
- Remove storing news item to db
- Remove `news`, `news_topics` and `news_tickers` tables
- Remove redundant models, tests
- Update frontend attributes
